### PR TITLE
README renderer workflow with RMarkdown README

### PR DESCRIPTION
**Pull request by cicdguy**
Adds a `README.Rmd` file which renders to a clean, formatted `README.md` upon push to `main, devel and pre-release`. This will allow for dynamic content to be added to the README.

### CODE REVIEW

**Review by bundfussr — APPROVED** [review]

**Comment by rossfarrugia** [comment]
@dinakar29 @bms63 i just changed Friday -> Monday for the admiral release schedule piece as discussed and agreed in the chatroom. if all OK happy for you to merge this one anytime.

I couldn't find anywhere else we talk about the Friday schedule but please update if you spot anywhere (FYI @thomas-neitmann too)

**Comment by rossfarrugia** [comment]
Ah the links all need updating now to take out the "main" bit until we put Dinakar's suggestion back in for admiral core (maybe for next release?).

**Review by bms63 — COMMENTED** [review]
Why are we wiping out most of the WORDLIST for the template?

File: inst/WORDLIST [hunk 1]
```diff
@@ -1,86 +1,33 @@
...
-Fridericia's
-Fujimoto
-functions’
-funder
-Gehan
-GitHub
-GlaxoSmithKline
-groupwise
+CMD
+Chagnelog
```
Comment (bms63) [inline inst/WORDLIST]: I would think this is something we want flagged?

File: inst/WORDLIST [hunk 2]
```diff
@@ -1,86 +1,33 @@
...
+anonymised
+cdisc
+dev
+github
+https
+onboarding
+pharmaverse
+prioritise
+renv
+reolve
```
Comment (bms63) [inline inst/WORDLIST]: Is this one used somewhere?

File: README.Rmd [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+1.  Once repo has been created click Green Code button and download the repo using `https` or `ssh`
+1.  Run `renv::restore()` - you will see a prompt and this will take a few minutes
+     - This will ensure that your development environment for your extension package is        synced with other admiral packages
+1.  Update/Remove the following files to use your extension name:
+    - `DESCRIPTION` File - Name, Authors
+    - `admiralext.Rproj`
+    - `testthat.R`
+    - `News.md`
+1.  Run `devtools::load_all()` and resolve issues
+1.  Run `devtools::document()` and reolve issues
```
Comment (bms63) [inline README.Rmd]:
```suggestion
1.  Run `devtools::document()` and resolve issues
```

File: inst/WORDLIST (hunk 1 above)
Comment (rossfarrugia) [reply inst/WORDLIST]: agree - thats a typo somewhere!

File: inst/WORDLIST (hunk 2 above)
Comment (bms63) [reply inst/WORDLIST]: found it and fixed

Comment (rossfarrugia) [reply inst/WORDLIST]: typo again

File: inst/WORDLIST (hunk 1 above)
Comment (bms63) [reply inst/WORDLIST]: Do you mind fixing this in the News.md file and removing this from the wordlist?

Comment (rossfarrugia) [reply inst/WORDLIST]: done

**Comment by rossfarrugia** [comment]
@dinakar29 from what you say above, i guess if we merge this to main then `README.md` will then render and all the broken links get fixed right?

**Comment by bms63** [comment]
Can we get back the original wordlist?  I think this is handy for admiral extension packages.

**Comment by rossfarrugia** [comment]
@bms63 would be hard to maintain that here as every package ext wordlist will evolve with new words over time? i think we'd be better just using the automation and running the script each time that adds the accepted words to the wordlist for you in your package repo rather than trying to maintain a master list here for new packages

**Comment by bms63** [comment]
> @bms63 would be hard to maintain that here as every package ext wordlist will evolve with new words over time? i think we'd be better just using the automation and running the script each time that adds the accepted words to the wordlist for you in your package repo rather than trying to maintain a master list here for new packages

I was just thinking that the original admiral wordlist would be the starter wordlist for the extension packages.  I don't think we should maintain it overtime either.  Just nice to have one out of the box that is comprehensive for our specific industry.

No problem if you want to just carry on as is.

**Comment by cicdguy** [comment]
> @dinakar29 from what you say above, i guess if we merge this to main then `README.md` will then render and all the broken links get fixed right?

Yes, that is correct.

**Comment by rossfarrugia** [comment]
> I was just thinking that the original admiral wordlist would be the starter wordlist for the extension packages. I don't think we should maintain it overtime either. Just nice to have one out of the box that is comprehensive for our specific industry.
> 
> No problem if you want to just carry on as is.

Yeh i would just leave it as is Ben, each package ext dev team can maintain using the automated check and it's a quick line to run to get things added.

**Review by rossfarrugia — APPROVED** [review]

**Comment by rossfarrugia** [comment]
@dinakar29 it didn't work after the merge, i forward'ed you the failed workflow.

**Comment by cicdguy** [comment]
@rossfarrugia  The version of `pkgdown` in the current `renv.lock` is old. My subsequent PR should fix it: https://github.com/pharmaverse/admiraltemplate/pull/26/files#diff-0a4513e8dc7e92cfc41839907ae36b739c44d06cd3323735b29423ce4b8c9887R489-R490

Screen Shot 2022-07-11 at 6 04 08 AM

**Pull request merged**